src/main/app/case/CosApiClient.ts

```diff
@@ -130,15 +130,19 @@ export class CosApiClient {
         partyType,
         caseType,
       };
-      const response = await Axios.post(config.get('services.cos.url') + `/${caseId}/${eventName}/case-update`, data, {
-        headers: {
-          Accept: 'application/json',
-          'Content-Type': 'application/json',
-          Authorization: 'Bearer ' + user.accessToken,
-          ServiceAuthorization: 'Bearer ' + getServiceAuthToken(),
-          accessCode: 'Dummy accessCode',
-        },
-      });
+      const response = await Axios.post(
+        config.get('services.cos.url') + `/${caseId}/${eventName}/case-update-2`,
+        data,
+        {
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+            Authorization: 'Bearer ' + user.accessToken,
+            ServiceAuthorization: 'Bearer ' + getServiceAuthToken(),
+            accessCode: 'Dummy accessCode',
+          },
+        }
+      );
```

Comment:
this will not work

Reply:
apologies @AlokDatta for that , yeah reverted back to case-update method now !! kindly review it ...